.github/workflows/build_and_test.yml

```
@@ -199,6 +199,23 @@ jobs:
       build-tools-image: ${{ needs.build-build-tools-image.outputs.image }}-bookworm
     secrets: inherit
 
+  validate-compute-manifest:
+    runs-on: ubuntu-22.04
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install jsonschema
+        run: npm install -g @sourcemeta/jsonschema
```

Need to install a particular version of a package:
```suggestion
        run: npm install -g @sourcemeta/jsonschema@9.3.4
```